fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_frenzy.dm

```diff
@@ -39,9 +39,12 @@
 	var/was_tooluser = FALSE
 	/// The stored Bloodsucker antag datum
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum
+	///Boolean indicating whether or not the owner is part of the Brujah clan
+	///Used in passive burn damage calculation
+	var/brujah = FALSE
```

Comment:
im actually gonna do it

Reply:
This goes against the whole thing of clans being datums and are modular, being able to be easily removed/added. It was the benefit to remaking the entire clan system in https://github.com/fulpstation/fulpstation/pull/745

Reply:
Should I perhaps remove this and replace the damage portion of `/datum/status_effect/frenzy/tick()` with a signal along the lines of `BLOODSUCKER_DAMAGED_BY_FRENZY`?